fvm/transactionEnv.go

```diff
@@ -735,6 +740,24 @@ func (e *TransactionEnv) GenerateUUID() (uint64, error) {
 }
 
 func (e *TransactionEnv) meterComputation(kind common.ComputationKind, intensity uint) error {
```

Comment:
This is called very often.

I'm not certain of the time complexity of doing `select { case <-e.executionCtx.Done(): ...` every-time for transactions where this is never actually needed in a transaction...

Can I suggest packaging this into a separate function. Maybe something like:
```
func (e *TransactionEnv) meterComputation(kind common.ComputationKind, intensity uint) error {
  err := e.ExecutionTerminationError()
  if err != nil {
      return err
  }
  //...
}
```

The Transaction `ExecutionTerminationError` would just be a noop